packages/tasks/src/hardware.ts

```diff
@@ -489,6 +489,9 @@ export const SKUS = {
 			"Intel Core Ultra 7 265KF": {
 				tflops: 1.53,
 			},
+			"Intel Core 14th Generation (i7)": {
+				tflops: 1.13,
```

Comment:
Hi @1TSnakers - Thanks for the PR where are you getting these values for? 

I get 0.8 TFLOP (ref: https://nanoreview.net/en/cpu/intel-core-i7-14700hx)